src/main/java/com/launchdarkly/testhelpers/httptest/Handlers.java

```diff
@@ -231,27 +224,6 @@ public static Handler hang() {
     };
   }
 
-  /**
-   * Creates a {@link Handler} that causes the server to return an invalid HTTP response, which
-   * should be seen by the client as a {@link ProtocolException}. Use this to test the behavior
-   * of client logic that needs to handle exceptions differently from HTTP error statuses.
-   *  
-   * @return a {@link Handler}
-   * @deprecated This method is deprecated because the mechanism for forcing the HTTP server to
-   * close the connection early is fragile and relies on implementation details of the
-   * underlying server framework. A more reliable approach is to use a {@link TcpServer} instead
-   * of an {@link HttpServer}, and configure it to close the connection without a response.
-   */
-  @Deprecated
```

Comment:
Removing this deprecated method now because there's no way to do such a thing in nanohttpd.